fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/FunctionBinder.java

```diff
@@ -212,4 +215,17 @@ public Expression visitBetween(Between between, CascadesContext context) {
         Between newBetween = between.withChildren(rewrittenChildren);
         return TypeCoercionUtils.processBetween(newBetween);
     }
+
+    @Override
+    public Expression visitInSubquery(InSubquery inSubquery, CascadesContext context) {
+        Expression newCompareExpr = inSubquery.getCompareExpr().accept(this, context);
+        Expression newListQuery = inSubquery.getListQuery().accept(this, context);
+        ComparisonPredicate newCpAfterUnNestingSubquery =
+                new EqualTo(newCompareExpr, ((ListQuery) newListQuery).getQueryPlan().getOutput().get(0));
```

Comment:
there are some exceptions.
for example:  select * from T where T.a in (select b from B)
T.a is int column, and B.b is bitmap column.
the semantics of this sql is bitmap_contains( bitmap_union(B.b), T.a)

Reply:
done